spsdk/image/images.py

```diff
@@ -1089,8 +1089,11 @@ def parse(cls, stream: Union[bytes, bytearray, BufferedReader, BytesIO],
         # Parse CSF
         if obj.ivt.csf_address:
             csf_start = start_index + (obj.ivt.csf_address - obj.ivt.ivt_address)
-            obj.csf = SegCSF.parse(read_raw_segment(stream, SegTag.CSF, csf_start))
-            # obj.csf.padding = csf_start + obj.csf.size
+            try:
+                obj.csf = SegCSF.parse(read_raw_data(stream, cls.CSF_SIZE, csf_start))
+                obj.csf.padding = cls.CSF_SIZE - obj.csf.size
+            except NotEnoughBytesException:
+                pass
```

Comment:
Would it make sense to set the `obj.ivt.csf_address` to 0, when the CSF is missing (sort of "fixing" the pre-populated IVT)?

Reply:
I think we should leave `obj.tvt.csf_address` as it is. It could be used to `export()` the CSF at the specified offset instead, for example if one thinks one needs to set it to some particular value.

Reply:
OK